web/pdf_page_view.js

```diff
@@ -1068,7 +1068,10 @@ class PDFPageView {
         await this.#finishRenderTask(renderTask);
 
         if (this.textLayer || this.annotationLayer) {
-          this.structTreeLayer ||= new StructTreeLayerBuilder(pdfPage);
+          this.structTreeLayer ||= new StructTreeLayerBuilder(
+            pdfPage,
+            this.viewport.rawDims
```

Comment:
Nit: Can we use the local variable, given that we only need to access `rawDims` here?
```suggestion
            viewport.rawDims
```